Cargo.toml

```diff
@@ -339,7 +339,7 @@ test-fuzz = "3.0.4"
 thiserror = "1.0.40"
 # tokio is defined at a specific version to support patching with iota-sim
 # Do not upgrade without updating https://github.com/iotaledger/iota-sim first
-tokio = "=1.43.0"
+tokio = "=1.44.2"
```

Comment:
How come sui doesn't have an `=` here?

Reply:
Not sure, because if someone bumps is on accident it will break simtests